docs/content/querying/sql.md

```diff
@@ -65,28 +68,27 @@ on broker startup and also periodically in the background through
 [SegmentMetadata queries](../querying/segmentmetadataquery.html). Background metadata refreshing is triggered by
 segments entering and exiting the cluster, and can also be throttled through configuration.
 
-This cached metadata is queryable through the "metadata.COLUMNS" and "metadata.TABLES" tables. When
-`druid.sql.planner.useFallback` is disabled (the default), only full scans of this table are possible. For example, to
-retrieve column metadata, use the query:
+This cached metadata is queryable through the "INFORMATION_SCHEMA.TABLES" and "INFORMATION_SCHEMA.COLUMNS" tables. Druid
```

Comment:
The fields we include do follow the standard, but we don't include every field. I added a doc about the fields we offer.